Assets/Scripts/Robo.cs

```diff
@@ -15,13 +15,23 @@ public class Robo : MonoBehaviour
     public float max_speed = 12;
     public float decay_rate = 25;
 
+    // Player health system variables
+    public float invincibility_duration_seconds = 1.5f;
```

Comment:
I think we should simplify name, i.e. `invincibility_duration_seconds-->invincibility_duration`

Reply:
I chose the name because I wanted to specify that it was in seconds so that if someone wants to tweak the value in the Inspector, they don't have to dig through the C# script to figure out what the units are nor use trial and error to guess the units.

It adds information that will likely speed up variable fine-tuning later.

Reply:
I think it is fine to assume seconds as a standard unit of time and mark exceptions rather than keep everything verbose. Thoughts?

Reply:
Alright, I did some brief research, and it *looks* like all the major time-related functions in Unity default to seconds for their units, so I think I'm comfortable with that as the standard. I'll change the variable name and push the change soon, but I have a new question.

Should we codify somewhere that this is a standard in our code so that we don't forget later? Like maybe have a text file in the repository listing standards we've chosen or a code-standards text channel in the discord.

Reply:
Pushed the change, see question from previous message before resolving conversation.